tutorials/roofit/rf102_dataimport.py

```diff
@@ -156,5 +41,109 @@ def makeTTree():
     return tree
 
 
-if __name__ == "__main__":
-    rf102_dataimport()
+############################
+# Importing ROOT histograms
+############################
+# Import ROOT TH1 into a RooDataHist
+# ---------------------------------------------------------
+# Create a ROOT TH1 histogram
```

Comment:
I am afraid these six lines will not look good in a notebook. If you are already familiar with the notebook interface, what you can do is copy these lines into a cell of type markdown in a notebook, and then remove the first "#" character from each line. This is exactly what will happen in the translation into a notebook.